Resources.html

```diff
@@ -81,25 +101,65 @@
+<div class="container-fluid text-center" style="height: 100%; padding-top: 50px; padding-bottom: 60px;">    
+    <div class="col-sm-10 text-center"></div>
+      <h1>Getting Started</h1>
+      <h4>Cybersecurity encompasses so many areas that it's hard to know where to start. Here are some of our recommendations for learning and practicing the basics when it comes to hacking the web and beyond!</h4>
       <hr>
-      <h3>Test</h3>
-      <p>Lorem ipsum...</p>
-    </div>
-  </div>
+      <div class="col-sm-6 col-md-6 col-lg-6 text-left">
+          <div class="row">
+            <div class="div-highlight">
+            <h3>Where to practice</h3>
+            </div>
+            <a href="https://overthewire.org/wargames/bandit/" target="_blank"><b>Over the Wire</b></a><p>A free series of wargames starting with "Bandit" that help you learn and practice security concepts in successive stages that build on each other.</p>
+            <a href="https://www.hackthebox.com/" target="_blank"><b>Hack the Box</b></a><p>A challenging platform that offers free and vulnerable machines to scan, explore, and hack in a CTF style.</p>
+            <a href="https://portswigger.net/" target="_blank"><b>Port Swigger</b></a><p>A free web security based platform that includes a web academy to learn almost every concept regarding website hacking and practice in vulnerable labs. Burp Suite is the main tool used for these labs, and there is a free community edition for learning purposes.</p>
+            <a href="https://tryhackme.com/" target="_blank"><b>Try Hack Me</b></a><p>Free training platform that has individual rooms to learn each security concept as it relates to web, cryptography, Windows, Linux, and software like Wireshark, Hydra, and Nessus. Similar to Hack the Box, but it is considered easier and better for beginners.</p>
+            <a href="https://cyberdefenders.org/" target="_blank"><b>Cyber Defenders</b></a><p>A training platform with a focus on defending networks from malicious attacks.</p> 
+            <a href="https://blueteamlabs.online/" target="_blank"><b>Blue Team Labs</b></a><p>A similar platform to train in defensive techniques related to security investigations.</p>
+          </div>
+          <div class="row"> 
+            <div class="div-highlight">
+            <h3>Where to learn Linux</h3>
+            </div>
+            <a href="https://linuxjourney.com/" target="_blank"><b>Linux Journey</b></a><p>An easy-to-follow guide on everything you should know about Linux that comes with helpful examples to reinforce your knowledge.</p>
+            <a href="https://www.linuxcommand.org/" target="_blank"><b>Linux Command</b></a><p>Focuses mainly on terminal commands, shell scripts, and learning to get a handle on optimizing your experience using Linux.</p>
```

Comment:
Good description but what is that, if you could just specify that at the beginning that would be good (i.e. a guide, a wiki, etc.).